swift/container/backend.py

```diff
@@ -1196,7 +1213,7 @@ def pivot_nodes_to_items(self, nodes):
     def build_pivot_ranges(self):
         ranges = list()
 
-        for node in self.get_pivot_ranges():
+        for node in sorted(self.get_pivot_ranges()):
```

Comment:
Hmm, well what is the right order.. i guess that's the first question. Order by lower.. or do we need to order by lower and then by upper. 

I guess what I'm asking what's wrong with the current order, what have you found in your research? Cause yeah, sorted here does seem a little _interesnting_ when we are just grabbing it from the DB.